cloud_experts_tutorials/cloud-experts-consistent-egress-ip.adoc

```diff
@@ -261,17 +289,23 @@ $ oc run \
   --image=registry.access.redhat.com/ubi9/ubi -- \
   bash
 ----
-
-Once inside the pod, you can send a request to the load balancer, ensuring that you can successfully connect:
-
++
+. Send a request to the load balancer and ensure that you can successfully connect:
++
 [source,terminal]
 ----
 $ curl -s http://$LOAD_BALANCER_HOSTNAME
 ----
-
-You should see output similar to the following, indicating a successful connection. It should be noted that the `client_address` below is the internal IP address of the load balancer rather than our egress IP. Successful connectivity (by limiting the service to `.spec.loadBalancerSourceRanges`) is what provides a successful demonstration:
-
++
+. Check the output for a successful connection: 
++
+[NOTE]
+====
+The `client_address` is the internal IP address of the load balancer not your egress IP. Successfully connecting with your service limited to `.spec.loadBalancerSourceRanges` is a successful outcome.
```

Comment:
This note is a little weird, maybe since the second sentence uses successful twice. Maybe "You can verify that you have successfully configured the client address by connecting with your service..."